RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/internal/api/Api.java

```diff
@@ -31,14 +27,14 @@
 
     private final String brandName;
 
-    public Api(@NonNull Context applicationContext, @NonNull String tenantId, @NonNull String brandName)
+    public Api(@NonNull Context applicationContext, @NonNull String tenantId, @NonNull String brandName, int timeoutInSeconds)
```

Comment:
wcześniej nie było timeoutu?

Reply:
Wcześniej timeout był już odpalany na całość w momencie dodawania do widoku GDPR view bo to się działo na starcie już.

Teraz mamy strzały api jeszcze przed pokazaniem view, dlatego dodałem timeout na nie tak jak na ios. 

Swoją drogą średnio mi się podoba to podejście, że temeout jest na request a nie na całość działania.
Może być sytuacja, że pierwszy request będzie trwał 9 sekund, kolejny 9 i potem wyświetlanie planszy 9 i mamy ekran za 27 sekund. No ale rozumiem, że jest to uproszczenie bo sytuacja raczej skrajna.